content/quickstart.md

```diff
@@ -345,3 +345,97 @@ and sends invoices to clients on the same schedule.
 
 [harvest]: https://www.getharvest.com/
 [accounts]: mailto:accounts@oddbird.net
+
+## Success Criteria:
+
+This guide will help you understand the expectations for your role at OddBird.
+The criteria are divided into three groups: Work Quality, Communication, and
+Learning and Growth. Each group has three levels of performance: Under performs,
+Meets, and Exceeds. You can use this guide to assess your own performance,
+identify your strengths and areas for improvement, and set your goals and action
+plans.
```

Comment:
I came up with these three arbitrary groups, what do you think? Also, is the current "list" style more appropriate than a table? Or plain prose?

Reply:
These groups make sense to me! I really like the list style and find it easy to read/comprehend quickly.